### ISSUE TITLE
Search error: Failed to get files list: path should be a `path.relative()`d string

### ISSUE THREAD

**Issue by shayandavoodii** [codeque-co/codeque#6]
Hi,
I wanted to test this great extension, but it throws an error when I want to search:  
`Search error: Failed to get files list: path should be a `path.relative()`d string, but got "E:\AUT\Thesis\.ipynb_checkpoints"`  
How should I fix it?  

I even tried the following way but it throws the same error:  
![image](https://user-images.githubusercontent.com/52105833/210366942-96831ee6-9725-422e-b3b2-0ff380934bed.png)

**Comment by jayu** [comment]
Hey, Just wanted to confirm two things
- Are you using Windows? 
- Does the search crash without any filters in files list ? Like after fresh install ? Or it crashes only if you use searching in forlder?

**Comment by shayandavoodii** [comment]
> Hey, Just wanted to confirm two things
> - Are you using Windows? 
> - Does the search crash without any filters in files list ? Like after fresh install ? Or it crashes only if you use searching in forlder? 
> 

1. Yes
2. It doesn't show me any results with/without applying filters.
3. I used it to search through a directory that contains .py files!

**Comment by jayu** [comment]
3. That's important consideration :D Are there any `js|jsx|ts|tsx|json|mjs|cjs` files in that directory? 
CodeQue supports only JavaScript, Typescript and JSON for now, more langs will be added in future

Nevertheless, the error that you've encounter with relative path should not happen, It's perhaps some edge case on Windows that I didn't captured. I will work on fixing this in next week. 
This issue is perhaps related to #8

**Comment by shayandavoodii** [comment]
> Are there any js|jsx|ts|tsx|json|mjs|cjs files in that directory?

No. :(  
Thank you.

**Comment by jayu** [comment]
He @shayandavoodii unfortunately I could not reproduce that issue 😞 .

I've added stack trace to error messages in release `0.20.1`. Could you try to reproduce the issue one more time and send me full error message screenshot? I will try to figure out where it's actually failing.

Also, I've added support for any programming language in `text` mode. Text mode does not allow for structural search, but allow for multiline / formatting agnostic search and easy wildcards usage. I hope you will be able to benefit from it to search your Python code!

**Comment by shayandavoodii** [comment]
> He @shayandavoodii unfortunately I could not reproduce that issue 😞 .
> 
> I've added stack trace to error messages in release `0.20.1`. Could you try to reproduce the issue one more time and send me full error message screenshot? I will try to figure out where it's actually failing.
> 
> Also, I've added support for any programming language in `text` mode. Text mode does not allow for structural search, but allow for multiline / formatting agnostic search and easy wildcards usage. I hope you will be able to benefit from it to search your Python code!

Hi, Sorry for the late answer. Sure!
```
Search error: Failed to get files lists: path should be a `path.relative()`d string, but got "E:\AUT\Thesis\.ipynb_checkpoints"

Stack:
RangeError: path should be a `path.relative()`d string, but got "E:\AUT\Thesis\.ipynb_checkpoints"
    at f (c:\Users\Shayan\.vscode\extensions\codeque.codeque-0.20.1\dist\extension.js:23:8265082)
    at _ (c:\Users\Shayan\.vscode\extensions\codeque.codeque-0.20.1\dist\extension.js:23:8265131)
    at g._test (c:\Users\Shayan\.vscode\extensions\codeque.codeque-0.20.1\dist\extension.js:23:8266427)
    at g.ignores (c:\Users\Shayan\.vscode\extensions\codeque.codeque-0.20.1\dist\extension.js:23:8266664)
    at c:\Users\Shayan\.vscode\extensions\codeque.codeque-0.20.1\dist\extension.js:23:8266733
    at Array.filter (<anonymous>)
    at g.filter (c:\Users\Shayan\.vscode\extensions\codeque.codeque-0.20.1\dist\extension.js:23:8266766)
    at t (c:\Users\Shayan\.vscode\extensions\codeque.codeque-0.20.1\dist\extension.js:23:8189752)
    at t.getFilesList (c:\Users\Shayan\.vscode\extensions\codeque.codeque-0.20.1\dist\extension.js:23:8190151)
    at async Promise.all (index 0)
```
Note that I passed the following steps for it:
1. Opened the cq tab
2. exact mode activated
3. search for `import cr` in the search field
4. hit the search button

Thanks for adding the text mode!

**Comment by jayu** [comment]
Thanks @shayandavoodii I will work on this as first thing in my next CodeQue programming session, I will get back to you 😀

**Comment by jayu** [comment]
Hey @shayandavoodii the bug should be fixed in CodeQue extension in version `0.20.2` 🍾 

Thank you for that bug report, it was quite important one and quite stupid mistake :D And thank you for patience

**Comment by shayandavoodii** [comment]
Hey @jayu. Thanks! It's super helpful. I look forward to adding new languages.  
![image](https://user-images.githubusercontent.com/52105833/216756984-bb63498f-36bc-46ab-a136-0e574aa65849.png)
